From 9c61256d5f92950d7c8a96a7c17ef5a8bd911006 Mon Sep 17 00:00:00 2001
From: Travis Reitter <travis.reitter@collabora.co.uk>
Date: Tue, 27 Aug 2013 17:07:14 +0000
Subject: Set the GIR namespace and version for folks-generics

This ends up avoiding a valadoc crasher (bgo#706857) which the maintainers
can't reproduce.  Strictly speaking, we should have it in place anyhow, so
it should be pretty safe.

It also ends up passing these details along to folks-internal.vapi when it's
generated as well.
---
diff --git a/folks/folks-generics.vapi b/folks/folks-generics.vapi
index 867257f..0f55c98 100644
--- a/folks/folks-generics.vapi
+++ b/folks/folks-generics.vapi
@@ -26,6 +26,7 @@
  * GIR to Vala loses the generic types. FIXME: GNOME #639908 would
  * make it possible to go via GIR like tests/lib/telepathy/contactlist does. */
 
+[CCode (gir_namespace = "Folks", gir_version = "0.6")]
 namespace Folks
 {
   [CCode (cheader_filename = "folks/small-set.h")]
--
cgit v0.9.2
